models/classes/LtiLaunchData.php

```diff
@@ -146,6 +150,7 @@ public static function fromLti1p3MessagePayload(
         LtiMessagePayloadInterface $ltiMessagePayload,
         PlatformInterface $platform = null
     ): self {
+        // phpcs:disable Generic.Files.LineLength
```

Comment:
Why you do not refactor this?

Reply:
too much to just simply break into multiple lines, do not see much sense in it,
at once no general idea how to refactor it properly, do you have one?

Reply:
Just split it in more lines like on image (couldn write it in markdonw :| ) 
![image](https://user-images.githubusercontent.com/118974926/213226788-7c361ff9-4846-47c7-b338-6c3f1b0d4fa0.png)